runtime/test/manifest-test.js

```diff
@@ -38,14 +38,24 @@ describe('manifest', function() {
   it('can parse a manifest containing a particle specification', async () => {
     let manifest = await Manifest.parse(`
       schema Product
-      particle Chooser in 'chooser.js'
-        Chooser(in [Product] choices, out [Product] resultList)
-
-        renders: action(resultList) none-need
-
-        Description {
-          pattern: Choose from \${choices}
-        }`);
+      schema Person
+      particle TestParticle in 'testParticle.js'
+        TestParticle(in [Product] list, out Person person)
+        affordance dom
+        affordance dom-touch
+        must consume root
+          formFactor big
+          provide action
+            view list
+            formFactor big
+          provide preamble
+            formFactor medium
+          provide annotation
+        consume other
+        description
+          pattern \`hello world \${list}\`
```

Comment:
is 'pattern' a special name? If yes, and there's just one, perhaps this should be on the same line as description:

```
  description `hello world ${list}`
    list `my special list`
```